### ISSUE TITLE
Throuws error when setup.py install

### ISSUE THREAD

**Issue by justinkotzebue** [DHI-GRAS/gdal_utils#1]
https://github.com/DHI-GRAS/gdal_utils/blob/00f261894b47445e371899cb4a0730666ffb7eb5/gdal_utils/gdal_scripts/gdal_rasterize.py#L109

I think it should look like this:
`raise RuntimeError('\'%s\' does not exist in the file system.') `

**Comment by j08lue** [comment]
Exactly. The failing syntax only works with Python 2. See http://python-future.org/compatible_idioms.html#raising-exceptions
